assignments/prototypes.js

```diff
@@ -102,9 +138,29 @@
   console.log(archer.greet()); // Lilith offers a greeting in Elvish.
   console.log(mage.takeDamage()); // Bruce took damage.
   console.log(swordsman.destroy()); // Sir Mustachio was removed from the game.
-*/
+
 
   // Stretch task: 
   // * Create Villain and Hero constructor functions that inherit from the Humanoid constructor function.  
+
+  function Villain (attr) {
+    this.evilPower = attr.evilPower;
+    this.menacingLaugh = attr.menacingLaugh;
+    Humanoid.call(this, attr)
+  }
+
+  Villain.prototype = Object.create(Humanoid.prototype)
+
+  function Hero(attr) {
+    this.heroPower = attr.heroPower;
+    this.heartWarmingLaugh = attr.heartWarmingLaugh;
+    Villain.call(this, attr)
+  }
+
+  Hero.prototype = Object.create(Villain.prototype)
+
+
```

Comment:
Why not add cool Hero moves here? Like maybe add a method to the prototype, to receive the key to the city from the mayor when the Hero wins?